src/connector.ts

```diff
@@ -134,4 +142,22 @@ export class Connector {
 
 		addedChanged(unprotectString(this.coreHandler.core.deviceId))
 	}
+
+	private setupReloadDataKoaEndpoint(): void {
+		const KOA_PORT: number = 3007
+		const RUNDOWN_EXTERNAL_ID_SUFFIX: string = '_1'
+
+		this.koaRouter.get('/reloadData/:rundownName', async (ctx, next): Promise<void> => {
+			if (!this.iNewsFTPHandler.iNewsWatcher) {
+				return
+			}
```

Comment:
In the case where we can't reload the data, wouldn't the caller be interested in knowing that the rundown has not been reloaded?

Reply:
I have added more handling towards this purpose!